cmd/manager/main.go

```diff
@@ -91,7 +91,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	clientset, err := v1alpha1.GetClientset()
+	clientset, err := v1alpha1.GetClientsetFromConfig(cfg)
```

Comment:
In my opinion you can also delete `GetClientsetFromConfig` and move `clientset, err := kubernetes.NewForConfig(config)` here. I really don't see a point of having a function that only calls one other function.

Reply:
I changed it to `kubernetes.NewForConfig(cfg)`.